nomos-services/api/src/http/da_shares.rs

```diff
@@ -51,65 +49,68 @@ where
         + Sync
         + Display
         + AsServiceId<StorageService<RocksBackend<StorageOp>, RuntimeServiceId>>,
+    // Service and storage layer types conversions
+    <DaStorageBackend<StorageOp> as StorageDaApi>::BlobId: From<DaShare::BlobId>,
```

Comment:
This is not the way to fit generics. There is no need to use `From`, both types should be the same.